python/example/network_ring.py

```diff
@@ -1,44 +1,53 @@
 import sys
 import arbor
-import matplotlib.pyplot as plt
+import pandas, seaborn
+from math import sqrt
 
 # Construct a cell with the following morphology.
 # The soma (at the root of the tree) is marked 's', and
 # the end of each branch i is marked 'bi'.
 #
-#         b2
+#         b1
 #        /
-# s----b1
+# s----b0
 #        \
-#         b3
+#         b2
 
 def make_cable_cell(gid):
-    b = arbor.flat_cell_builder()
+    # Associate labels to tags
+    labels = arbor.label_dict()
+    labels['soma'] = '(tag 1)'
+    labels['dend'] = '(tag 2)'
+
+    # Build a segment tree
+    tree = arbor.segment_tree()
+
+    # Soma (tag=1) with radius 6 μm, modelled as cylinder of length 2*radius
+    s = tree.append(arbor.mnpos, arbor.mpoint(-12, 0, 0, 6), arbor.mpoint(0, 0, 0, 6), tag=1)
 
-    # Soma with radius 6 μm, modelled as cylinder of length 2*radius
-    s  = b.add_cable(length=12, radius=6, name="soma", ncomp=1)
-    # Single dendrite of length 100 μm and radius 2 μm attached to soma.
-    b1 = b.add_cable(parent=s, length=100, radius=2, name="dend", ncomp=1)
-    # Attach two dendrites of length 50 μm to the end of the first dendrite.
+    # Single dendrite (tag=2) of length 100 μm and radius 2 μm attached to soma.
```

Comment:
tag 2 is typically associated with an axon, according to the SWC specs, tag 3 is dendrite. It doesn't matter here, but it's probably better to be consistent